packages/designto-token/token-container/index.ts

```diff
@@ -4,13 +4,15 @@ import { tokenizeBackground } from "../token-background";
 import { BoxShape } from "@reflect-ui/core/lib/box-shape";
 import { keyFromNode } from "../key";
 import { tokenizeBorder } from "../token-border";
+import { BoxShadowManifest } from "@reflect-ui/core";
+
 function fromRectangle(rect: nodes.ReflectRectangleNode): core.Container {
   const container = new core.Container({
     key: keyFromNode(rect),
     width: rect.width,
     height: rect.height,
     borderRadius: rect.cornerRadius,
-    boxShadow: rect.primaryShadow,
+    boxShadow: rect.shadows as BoxShadowManifest[],
```

Comment:
`as BoxShadowManifest[]` - necessary?
if true, explain why

Reply:
shadows return type is `ReadonlyArray<BoxShadowManifest>`.

A type conflict occurs.

`get shadows` root is `design-sdk/figma-reflect-node/base.node.ts`